### ISSUE TITLE
historytools.py reconcile crashes on temp to zero then suspend

### ISSUE THREAD

**Issue by scottleibrand** [loudnate/openaps-mmhistorytools#26]
pi@rpi4 ~/openaps-dev $ cat clean.json
```json
[
  {
    "timestamp": "2015-09-06T15:09:51",
    "_type": "PumpResume"
  },
  {
    "_type": "PumpSuspend",
    "_description": "PumpSuspend 2015-09-06T15:09:03 head[2], body[0] op[0x1e]",
    "timestamp": "2015-09-06T15:09:03",
    "_body": "",
    "_head": "1e00",
    "_date": "83490f060f"
  },
  {
    "_type": "TempBasalDuration",
    "_description": "TempBasalDuration 2015-09-06T15:09:02 head[2], body[0] op[0x16]",
    "timestamp": "2015-09-06T15:09:02",
    "_body": "",
    "_head": "1601",
    "duration (min)": 30,
    "_date": "82490f060f"
  },
  {
    "_type": "TempBasal",
    "temp": "absolute",
    "_description": "TempBasal 2015-09-06T15:09:02 head[2], body[1] op[0x33]",
    "timestamp": "2015-09-06T15:09:02",
    "_body": "00",
    "_head": "3300",
    "rate": 0.0,
    "_date": "82490f060f"
  }
]
```
```
pi@rpi4 ~/openaps-dev $ cat clean.json | openaps use munge reconcile
Traceback (most recent call last):
  File "/usr/local/bin/openaps-use", line 5, in <module>
    pkg_resources.run_script('openaps==0.0.6', 'openaps-use')
  File "/usr/local/lib/python2.7/dist-packages/setuptools-1.0-py2.7.egg/pkg_resources.py", line 492, in run_script

  File "/usr/local/lib/python2.7/dist-packages/setuptools-1.0-py2.7.egg/pkg_resources.py", line 1350, in run_script

  File "/usr/local/lib/python2.7/dist-packages/openaps-0.0.6-py2.7.egg/EGG-INFO/scripts/openaps-use", line 63, in <module>
    app( )
  File "/usr/local/lib/python2.7/dist-packages/openaps-0.0.6-py2.7.egg/openaps/cli/__init__.py", line 44, in __call__
    self.run(self.args)
  File "/usr/local/lib/python2.7/dist-packages/openaps-0.0.6-py2.7.egg/EGG-INFO/scripts/openaps-use", line 57, in run
    output = app(args, self)
  File "/usr/local/lib/python2.7/dist-packages/openaps-0.0.6-py2.7.egg/openaps/uses/__init__.py", line 92, in __call__
    return self.method.selected(args)(args, app)
  File "/usr/local/lib/python2.7/dist-packages/openaps-0.0.6-py2.7.egg/openaps/uses/__init__.py", line 31, in __call__
    return self.method(args, app)
  File "/usr/local/lib/python2.7/dist-packages/openaps-0.0.6-py2.7.egg/openaps/uses/use.py", line 26, in __call__
    output = self.main(args, app)
  File "/usr/local/lib/python2.7/dist-packages/openapscontrib.mmhistorytools-0.0.3-py2.7.egg/openapscontrib/mmhistorytools/__init__.py", line 205, in main
    tool = ReconcileHistory(*args)
  File "/usr/local/lib/python2.7/dist-packages/openapscontrib.mmhistorytools-0.0.3-py2.7.egg/openapscontrib/mmhistorytools/historytools.py", line 170, in __init__
    self.add_history_event(event)
  File "/usr/local/lib/python2.7/dist-packages/openapscontrib.mmhistorytools-0.0.3-py2.7.egg/openapscontrib/mmhistorytools/historytools.py", line 174, in add_history_event
    decoded = getattr(self, "_decode_{}".format(event["_type"].lower()))(event)
  File "/usr/local/lib/python2.7/dist-packages/openapscontrib.mmhistorytools-0.0.3-py2.7.egg/openapscontrib/mmhistorytools/historytools.py", line 217, in _decode_pumpresume
    new_event[key] = event[key]
KeyError: 'date'
pi@rpi4 ~/openaps-dev $
```

**Comment by loudnate** [comment]
This was fixed in 0.0.4, (https://github.com/loudnate/openaps-mmhistorytools/commit/e7f784a6956528e918ca5a7e910defa1bcd1a38b) which coincided with the change in decocare to remove date epochs. An upgrade will fix.

**Comment by scottleibrand** [comment]
Thanks.  I hate the fact that -U is not the default in easy_install like it is in apt etc: I thought I was installing openapscontrib.mmhistorytools fresh, but forgot I already had an old version installed.
